dev/AppNotifications/AppNotification.cpp

```diff
@@ -6,8 +6,15 @@ using namespace winrt::Windows::Data::Xml::Dom;
 
 namespace winrt::Microsoft::Windows::AppNotifications::implementation
 {
+    AppNotification::AppNotification()
+    {
+        THROW_HR_IF(E_NOTIMPL, !::Microsoft::Windows::PushNotifications::Feature_AppNotifications::IsEnabled());
```

Comment:
Not needed here. Just put this in the beginning of AppNotificationManager::GetDEfault() which is the real entry point to everything AppNotification related.

Reply:
I had added this into the ctor of each public runtime class, that's what push seems to be doing. 

I've removed it from all ctor (for toast) and only added one single check in AppNotificationManager::Default().